js/script.js

```diff
@@ -8,6 +8,51 @@
       });
   })
 
+  var npmService = [
+    'https://api.npmjs.org/downloads/point/last-day/gulp'
+  ].map(function (url) {
+    return fetch(url)
+      .then(function(resp) {
+        return resp.json();
+      });
+  })
+
+  var pluginService = [
+    'https://api.npms.io/v2/search?q=gulpplugin'
```

Comment:
@SomethingNew71 sorry to comment here again but it looks like you can be more specific with https://api.npms.io/v2/search?q=keywords%3Agulpplugin (just looked at their docs)